controllers/userController.ts

```diff
@@ -94,7 +94,7 @@ export async function getUserProfile(username: string): Promise<UserProfile> {
  * @param {string} opHours - The hours of operation of the bathroom (days and times)
  * @param {Optional<string>} description - Description of the bathroom
  * @param {string[]} tags - List of tags to filter the bathroom by
- * @param {Optional<Image>} images - Uploaded images of the bathroom
+ * @param {string} image - Uploaded images of the bathroom
```

Comment:
Similar to comment on profile picture, Kayla and I decided to match the backend for now. Kayla thinks that this would be better as a list of strings though, as there could be 0..many images associated. I agree.

Kayla, please create a follow up task for this as well.